queue_job/jobrunner/runner.py

```diff
@@ -219,16 +219,14 @@ def close(self):
 
     def _has_queue_job(self):
         with closing(self.conn.cursor()) as cr:
-            try:
-                cr.execute("SELECT 1 FROM ir_module_module "
-                           "WHERE name=%s AND state=%s",
-                           ('queue_job', 'installed'))
-            except psycopg2.ProgrammingError as err:
-                if unicode(err).startswith('relation "ir_module_module" '
-                                           'does not exist'):
-                    return False
-                else:
-                    raise
+            cr.execute("SELECT to_regclass('ir_module_module')")
+            [ir_module_module_exists] = cr.fetchone()
+            if not ir_module_module_exists:
+                return False
+            cr.execute(
+                "SELECT 1 FROM ir_module_module WHERE name=%s AND state=%s",
+                ('queue_job', 'installed')
+            )
```

Comment:
Well, the last version that is supported by PostgreSQL itself is 9.2.

I've changed the query to support older versions.